sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/StatefulProcessorHandleSuite.scala

```diff
@@ -48,7 +49,7 @@ class StatefulProcessorHandleSuite extends StateVariableSuiteBase {
       tryWithProviderResource(newStoreProviderWithStateVariable(true)) { provider =>
         val store = provider.getStore(0)
         val handle = new StatefulProcessorHandleImpl(store,
-          UUID.randomUUID(), keyExprEncoder, getTimeoutMode(timeoutMode))
+          UUID.randomUUID(), keyExprEncoder, TTLMode.NoTTL(), getTimeoutMode(timeoutMode))
```

Comment:
Is it possible to add unit testing to `StatefulProcessorHandleSuite` or `ValueSuite` to cover the TTL functionality at a UT level ?

Reply:
I added testcases for `StatefulProcessorHandleSuite`. Will add few more testcases for ValueStateImplWithTTL, and update the PR.

Reply:
Added UT testcases for `ValueStateImplWithTTL` in `ValueStateSuite`.